build-system/tasks/check-links.js

```diff
@@ -123,6 +123,8 @@ function checkLinksInFile(file) {
       // Localhost links don't work unless a `gulp` server is running.
       {pattern: /localhost/},
       // Templated links are merely used to generate other markdown files.
+      {pattern: /https:\/\/codepen.*/},
```

Comment:
Can the codepen links be fixed? Or are they expected to be broken? (If so, the preference is to put non-working links in backticks instead of adding them here.) 

Side note: the comment above this line looks incorrect.

Reply:
codepen is sending a 503 against the crawl, the link itself is working

Reply:
great catch - updated!

Reply:
> codepen is sending a 503 against the crawl, the link itself is working

You are correct. No difference between http and https. For posterity, here is the result from https://httpstatus.io.

![image](https://user-images.githubusercontent.com/26553114/101070970-ff89ef00-3569-11eb-8ca7-d0801695afc3.png)